backend/src/api/__init__.py

```
@@ -2,7 +2,8 @@
 LifeSync Personality Engine - API Module
 """
 
-from .server import app
+# Removing app import to avoid circular dependency
+# from .server import app
 from .config import config, Config
 
-__all__ = ['app', 'config', 'Config']
+__all__ = ['config', 'Config']
```

Ruff (0.15.2)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)